docs/src/schemaValidator.md.ejs

```diff
@@ -0,0 +1,87 @@
+## schemaValidator
+
+Unlike the rest of the ripple-lib API, schemaValidator is a static object on RippleAPI. It provides utility methods that do not use a server.
+
+## schemaValidate
+
+`RippleAPI.schemaValidator.schemaValidate(schemaName: string, object: any): void`
+
+This method checks an object for conformance to a specified schema. It does not return anything, but will throw a `ValidationError` if the object does not conform to the schema.
```

Comment:
What do you think about including an example for `schemaValidate`?

Reply:
Good idea